includes/fields-of-study.php

```diff
@@ -10,29 +10,34 @@ public function display_fields_of_study( $atts ) {
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=41746">Economics</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=32421">Education</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7786">Engineering and Computer Science</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7807">Foreign Languages and Cultures</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7775">Health Sciences</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=1862">History, Literature, and Philosophy</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=1861">Music</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7802">Physical and Mathematical Sciences</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=7839">Pre-Professional Studies</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=1876">Social Sciences</a></li>
-						<li class="fos-item"><a href="http://admission.wsu.edu/academics/fos/Public/area.castle?id=32423">Sport and Fitness</a></li>
-					</ul>
+				<h4>Find my field of study</h4>
+				<form title="Select a field of study by academic area">
+					<label>Select an Academic Area
+					<select>
+						<option value="7762">Agricultural Sciences</option>>
+						<option value="7774">Art, Architecture, and Design</option>>
+						<option value="7763">Biological and Environmental Sciences</option>>
+						<option value="41745">Business</option>>
+						<option value="41685">Business Programs Accredited by AACSB</option>>
+						<option value="1854">Communication</option>>
+						<option value="41746">Economics</option>>
+						<option value="32421">Education</option>>
+						<option value="7786">Engineering and Computer Science</option>>
+						<option value="7807">Foreign Languages and Cultures</option>>
+						<option value="7775">Health Sciences</option>>
+						<option value="1862">History, Literature, and Philosophy</option>>
+						<option value="1861">Music</option>>
+						<option value="7802">Physical and Mathematical Sciences</option>>
+						<option value="7839">Pre-Professional Studies</option>>
+						<option value="1876">Social Sciences</option>>
+						<option value="32423">Sport and Fitness</option>>
+					</select>
+					</label>
+					<input id="fos--select" value="View field of study" type="button" />
+				</form>
 			</div>
 		</div>
+
 		<div class="find-fos-alpha-wrapper">
 			<div class="find-fos-alpha">
 				<h3>Browse Alphabetically</h3>
```

Comment:
I think this makes sense under _Find My Field of Study_ so it probably should be a `<h4>`

Reply:
Agreed here too :)